.github/workflows/pull-request.yml

```diff
@@ -6,5 +6,5 @@ jobs:
   call_workflow:
     name: Run PR Build Workflow
     if: ${{ github.repository_owner == 'ballerina-platform' }}
-    uses: ballerina-platform/ballerina-standard-library/.github/workflows/pull-request-build-template.yml@main
+    uses: ballerina-platform/ballerina-standard-library/.github/workflows/pull-request-build-template.yml@dilanSachi-patch-1
```

Comment:
Can you use `workflow-test` branch here and check for build failures

Reply:
its working